### ISSUE TITLE
Using openstreetmap tiles but still getting google labels and logo

### ISSUE THREAD

**Issue by stefan-beyer** [react-community/react-native-maps#1631]
I am using custom tile server to display openstreetmap in react-native-maps.

```
<MapView
  region={{latitude: 48.117949,longitude: 7.853377,latitudeDelta: 0.0922,longitudeDelta: 0.0421,}}
  provider={null}
  rotateEnabled={false}
  style={{flex: 1}}
  showsUserLocation >
      <MapView.UrlTile urlTemplate="http://a.tile.openstreetmap.de/tiles/osmde/{z}/{x}/{y}.png" />
</MapView>
```

The tiles are displayed correctly.
But I still get sort of an overlay with city names from google maps and the google logo at the bottom left corner.
How can I remove google things completely from the map?

**Edit:** Google Map Tiles are still loaded too and are simply covered by the openstreatmap tiles.

![photo_2017-09-09_12-07-28](https://user-images.githubusercontent.com/26070175/30239222-ec6f065e-9557-11e7-930f-efc330bc8b9e.jpg)

**Comment by kbzowski** [comment]
+1

**Comment by keithtxw** [comment]
Specify mapType="none" as a prop in the MapView to hide the Google base map. 

Unfortunately that does not remove the logo. You could set a style to hide part of the map. Refer to this https://github.com/airbnb/react-native-maps/issues/1239

**Comment by Stophface** [comment]
I am using `<MapView.UrlTile .../> to add a `Custom Tile Overlay`.

```
<MapView 
   ...>
  <MapView.UrlTile
    urlTemplate={this.state.urlTemplate}
  />
</MapView>
```

 It sort of works. There are two problems:

1. When I pinch, drag and zoom in/out, it displays the underlaying tiles (on Android Google Maps). Once the pinching, dragging, zooming is done, it only then loads the `Tiles` passed with `<MapView.UrlTile .../>`. It looks like basically it loads two maps. First the underlaying, and then the one passed through `<MapView.UrlTile .../>`.
Here a Video: [https://www.file-upload.net/download-12828114/maps_prob1.mov.html](https://www.file-upload.net/download-12828114/maps_prob1.mov.html) (19.41 MB, download link)

**Comment by alvelig** [comment]
Generally is a duplicate of #773

**Comment by alvelig** [comment]
#1880

**Comment by pawan8525** [comment]
hi i am using below url of soil map tile
<MapView.UrlTile
urlTemplate=" ....../api/soil/wms/?x=5458&y=12859&z=15 " />

in x,y coordinate i pass converted lat long and in z i pass zoom level.
But my map overlay tile repeat, why i don't know?
Please help
